### DISCUSSION TITLE
fitting parameters for activity coeff mixing rules

### DISCUSSION THREAD

**Discussion by sonjaamsmith** [ClapeyronThermo/Clapeyron.jl#291] [Q&A] [answered]
Hello =]

Is there a way to fit the parameters for an activity coefficient-based mixing rule? I want to model some systems with Peng-Robinson using the Huron-Vidal mixing rules, but want to fit the parameters in the HV mixing rule to experimental data. I am aware of your example notebook on parameter estimation, but I'm not quite sure what I would have to put in the `toestimate` variable.

**Comment by pw0908** [comment, marked as answer]
Hi Sonja,

This is a really good question and one we didn't think of when making the examples. In short, it is possible to modify the activity coefficient model parameters in a GE mixing rule. When Clapeyron fits parameters, it goes through all parameters in the model struct, including the submodels, looking for parameters whose name matches those specified in `toestimate`. The only thing to be careful of is that none of the other submodels have parameters with the same name. I've updated the notebook to show how to do this.

**Reply by sonjaamsmith** [reply]
Cool thanks, I'll have a look.
